test/node.js

```diff
@@ -276,43 +264,77 @@ node.expectedFeeForTrsWithData = function (amount) {
 	return parseInt(node.fees.transactionFee) + parseInt(node.fees.dataFee);
 };
 
-// Returns a random username
+// Returns a random username of 16 characters
 node.randomUsername = function () {
```

Comment:
True random username would be dependent also from different length. Can you do sth like merge between letter part - `randomString.generate({length: node.randomNumber(1, 8), charset: 'alphabetic')`
and nonletter - `randomString.generate({length: node.randomNumber(1, 8), charset: '0123456789!@$&_.')`

Reply:
Done! Thanks @MaciejBaj